File: code/1035/1035.js
const input = require("fs").readFileSync("/dev/stdin", "utf8");
const lines = input.split(/\n|\s/);

const A = parseInt(lines[0]);
const B = parseInt(lines[1]);
const C = parseInt(lines[2]);
const D = parseInt(lines[3]);

if (B > C && D > A && C + D > A + B && C >= 0 && D >= 0 && A % 2 === 0) {
  console.log("Valores aceitos");
} else {
  console.log("Valores nao aceitos");
}

File: code/1035/1035.cs
using System;

class URI
{
    static void Main(string[] args)
    {
        int A, B, C, D;
        string[] input = Console.ReadLine().Split(' ');
        A = int.Parse(input[0]);
        B = int.Parse(input[1]);
        C = int.Parse(input[2]);
        D = int.Parse(input[3]);

        if (
            B > C && D > A && 
            (C + D) > (A + B) && 
            C > 0 && D > 0 && 
            A % 2 == 0
        )
        {
            Console.WriteLine("Valores aceitos");
        }
        else
        {
            Console.WriteLine("Valores nao aceitos");
        }
    }
}